test/docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM wpengine/golang as builder
+
+WORKDIR /
+
+# Install kubectl version 1.7.x. Currently seeing issues with 1.8, so stick with 1.7 for now.
+ENV KUBECTL_VERSION v1.7.9
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl
+RUN chmod +x ./kubectl
+
+FROM alpine
```

Comment:
The problem scratch has is that it was not allowing me to use kubectl as part of my path. So whenever lostromos made a kubectl call it would fail. Alpine didn't have that issue so I just went with it. The size doesn't seem to be much of an issue so I just left it with alpine.

Reply:
Ahh yes that would likely cause issues. Alpine is fine, it's nice and small.